consensus/src/context.rs

```diff
@@ -310,22 +333,49 @@ pub enum BlockChainContextRequest {
 }
 
 pub enum BlockChainContextResponse {
-    /// Blockchain context response.
+    /// A generic Ok response.
+    ///
+    /// Response to:
+    /// - [`BlockChainContextRequest::NewRXVM`]
+    /// - [`BlockChainContextRequest::Update`]
+    /// - [`BlockChainContextRequest::PopBlocks`]
+    /// - [`BlockChainContextRequest::ClearAltCache`]
+    /// - [`BlockChainContextRequest::AddAltChainContextCache`]
+    Ok,
+
+    /// Response to [`BlockChainContextRequest::Context`]
     Context(BlockChainContext),
+
+    /// Response to [`BlockChainContextRequest::CurrentRxVm`]
+    ///
     /// A map of seed height to `RandomX` VMs.
     RxVms(HashMap<usize, Arc<RandomXVm>>),
+
     /// A list of difficulties.
     BatchDifficulties(Vec<u128>),
+
+    /// Response to [`BlockChainContextRequest::HardForkInfo`]
+    HardForkInfo(HardForkInfo),
+
+    /// Response to [`BlockChainContextRequest::FeeEstimate`]
+    FeeEstimate(FeeEstimate),
+
+    /// Response to [`BlockChainContextRequest::AltChains`]
+    ///
+    /// If the inner [`Vec::is_empty`], there were no alternate chains.
+    AltChains(Vec<ChainInfo>),
+
     /// An alt chain context cache.
     AltChainContextCache(Box<AltChainContextCache>),
+
     /// A difficulty cache for an alt chain.
     AltChainDifficultyCache(DifficultyCache),
+
     /// A randomX VM for an alt chain.
     AltChainRxVM(Arc<RandomXVm>),
+
     /// A weight cache for an alt chain
     AltChainWeightCache(BlockWeightsCache),
-    /// A generic Ok response.
-    Ok,
 }
```

Comment:
### Signatures (blockchain context response)
Added to the existing `BlockChainContextResponse`.